deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: dhstore-data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Ti
+  storageClassName: io2
```

Comment:
we should use gp3 and bump its throughput as needed.

Reply:
Can I also ask you to start a snapshot on the dhstore on `dev` to get the ball rolling on switching to `gp3` there too.

Reply:
Sure, let me roll it out to prod first and get the ball rolling

Reply:
sounds good thanks Ivan